core/crypto/src/util.rs

```diff
@@ -59,15 +58,17 @@ impl<T1: Packable<Packed = [u8; 32]>, T2: Packable<Packed = [u8; 32]>> Packable
     type Packed = [u8; 64];
 
     fn unpack(data: &[u8; 64]) -> Option<Self> {
-        let (d1, d2) = array_refs!(data, 32, 32);
-        Some((unpack(d1)?, unpack(d2)?))
+        // TODO(mina86): Use split_array_ref once stabilised.
+        let d1 = unpack(data[..32].try_into().unwrap())?;
+        let d2 = unpack(data[32..].try_into().unwrap())?;
+        Some((d1, d2))
     }
 
     fn pack(&self) -> [u8; 64] {
         let mut res = [0; 64];
-        let (d1, d2) = mut_array_refs!(&mut res, 32, 32);
-        *d1 = self.0.pack();
-        *d2 = self.1.pack();
+        // TODO(mina86): Use split_array_mut once stabilised.
```

Comment:
We already have a polyfil here: https://github.com/near/nearcore/blob/1d09250875d534d10ad9d1e14513cff2479bd174/runtime/near-vm-logic/src/array_utils.rs#L6

As usual, it's not in a convenient place to re-use, not sure what would be the right crate to pull that into.